components/gfx/font_context.rs

```diff
@@ -9,7 +9,7 @@ use font_cache_thread::FontCacheThread;
 use font_template::FontTemplateDescriptor;
 use malloc_size_of::{MallocSizeOf, MallocSizeOfOps};
 use platform::font::FontHandle;
-use platform::font_context::FontContextHandle;
+pub use platform::font_context::FontContextHandle;
```

Comment:
I had to do this for the test, which doesn't seem ideal but I struggled to find an alternative. In the existing code, a `FontContext` instantiates a `FontContextHandle`, but instantiating a `FontContext` requires lots of dependencies (some of them external to this crate too).